src/adaptors/solend/index.js

```diff
@@ -0,0 +1,88 @@
+const fetch = require('node-fetch');
+
+const utils = require('../utils');
+
+const baseUrl = 'https://api.solend.fi';
+const configEndpoint = `${baseUrl}/v1/config`;
+const reservesEndpoint = `${baseUrl}/v1/reserves`;
+
+const TOKEN_LIST_URL = "https://cdn.jsdelivr.net/gh/solana-labs/token-list@main/src/tokens/solana.tokenlist.json"
+
+let tokenList
+
+async function getTokenList() {
+  if (!tokenList)
+    tokenList = (await utils.getData(TOKEN_LIST_URL)).tokens
+  return tokenList
+}
+
+const getCoingeckoId = async (address) => {
+  const tokenlist = await getTokenList();
+  return tokenlist.find((t) => t.address === address)?.extensions
+    ?.coingeckoId;
+}
+
+const assetToCoinGeckoIdMap = {
+  "USDT-USDC": "usd-coin",
+  "mSOL-SOL": "msol",
+  "BTC": "bitcoin",
+  "soETH": "ethereum",
+  "SBR": "saber",
+  "lsIN": "invictus",
+  "xSTEP": "step",
+  "PRT": "usd-coin",
+  "CASH": "usd-coin",
+  "UXD": "usd-coin",
+  "USDH": "usd-coin",
+  "GMT": "stepn",
+}
+
+const buildPool = async (reserveConfig, reserveData) => {
+  const liquidity = reserveData.reserve.liquidity;
+  const coingeckoId =  (await getCoingeckoId(liquidity.mintPubkey)) ?? assetToCoinGeckoIdMap[reserveConfig.asset] ?? reserveConfig.asset;
+  const apy = Number(reserveData.rates.supplyInterest) + reserveData.rewards.reduce((acc, reward) => reward.side === 'supply' ? Number(reward.apy) + acc : acc, 0)
+
+  const newObj = {
+    pool: reserveConfig.address,
+    secondaryString: reserveConfig.marketName.charAt(0).toUpperCase() + reserveConfig.marketName.slice(1) + ' Pool',
```

Comment:
Alternatively, we can simplify this by only showing our main pool assets for now if that's easier for you guys

Reply:
Hello 0xodia. cheers for the adaptor! 

I suggest we keep all pools, not just main. 
Instead of adding the pool name to the project field, i suggest to add it to the symbol field instead like
```
symbol: `${utils.formatSymbol(coingeckoId)} (${secondaryString})`
```

Also, I would not use coingecko Ids, but instead the symbols from `reserveConfig.asset` so we get `BTC` instead of `bitcoin`

Reply:
i see. i was worried about downstream code being dependent on using coingeckoId for consistency. For
[DefiLlama-Adapters](https://github.com/DefiLlama/DefiLlama-Adapters), that seemed important to get the correct price.